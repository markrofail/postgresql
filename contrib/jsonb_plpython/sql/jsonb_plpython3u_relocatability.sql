create extension jsonb_plpython3u cascade;
create schema test;
alter extension jsonb_plpython3u set schema test;
create function test.test(val jsonb) returns jsonb
language plpython3u
transform for type jsonb
as $$
return val
$$;

select test.test('1'::jsonb);

drop schema test cascade;
drop extension plpython3u cascade;
