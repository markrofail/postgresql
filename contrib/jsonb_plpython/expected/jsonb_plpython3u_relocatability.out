create extension jsonb_plpython3u cascade;
NOTICE:  installing required extension "plpython3u"
create schema test;
alter extension jsonb_plpython3u set schema test;
create function test.test(val jsonb) returns jsonb
language plpython3u
transform for type jsonb
as $$
return val
$$;
select test.test('1'::jsonb);
 test 
------
 1
(1 row)

drop schema test cascade;
NOTICE:  drop cascades to 2 other objects
DETAIL:  drop cascades to extension jsonb_plpython3u
drop cascades to function test.test(jsonb)
drop extension plpython3u cascade;
